packages/cf.js/src/app-factory.ts

```diff
@@ -57,11 +59,7 @@ export class AppFactory {
     const timeout = parseBigNumber(params.timeout, "timeout");
     const myDeposit = parseBigNumber(params.myDeposit, "myDeposit");
     const peerDeposit = parseBigNumber(params.peerDeposit, "peerDeposit");
-    try {
-      getAddress(params.peerAddress);
```

Comment:
It's not valuable to check whether these addresses are valid since the Node must do this anyway, and will likely return the exact same error as CFjs would here. Extra code complexity with no gain